lib/engines/hcktest/platforms/Win10_2004x86.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "Win10_2004x86",
+  "iso": "Win10_2004x86",
```

Comment:
As currently, we have separated client-specific options and platform options, and this ISO is a client iso only; update the name.